src/components/navigationLoader/NavigationLoaderExample.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { NavigationContainer } from './NavigationContainer';
+import { NavigationMenuSectionItem } from './navigationMenuSectionItem';
+import { NavigationMenuSection } from './NavigationMenuSection';
+
+export class NavigationLoaderExample extends React.Component<any, any> {
+  render() {
+    return (
+      <div className='mt2'>
+        <label className='form-control-label'>Navigation Loading</label>
+        <div className='flex flex-row flex-stretch'>
+          <NavigationContainer className={'full-content-y'}>
```

Comment:
Is this class really required for the sake of this example? I am not sure it does anything.

Furthermore, if you add a string property, you do not need to enclose it with brackets. Ex: `className='full-content-y'` will work as well.